packages/compass-connect/src/stores/index.js

```diff
@@ -682,8 +699,11 @@ const Store = Reflux.createStore({
 
   /**
    * Resets the connection after clicking on the new connection section.
+   *
+   * @param {String} connectionEntry The connection form entry
    */
-  onResetConnectionClicked() {
+  onResetConnectionClicked(connectionEntry) {
+    this.state.connectionEntry = connectionEntry;
```

Comment:
what is `connectionEntry`? i feel this name is a bit too generic here among all this `connectionModel`, `connection` .. etc, is it the type of connection (recent / favorite / new)?

Also can this ever be something different from `CONNECTION_ENTRY_NEW`?